time timestamp
uint32 arbitration_id
uint8[] data
